src/main/kotlin/com/chattriggers/ctjs/CTJS.kt

```diff
@@ -111,4 +121,27 @@ object CTJS {
         val connection = makeWebRequest(url)
         connection.getInputStream()
     }
+    private val sslContext by lazy {
+        try {
+            val myKeyStore = KeyStore.getInstance("JKS")
+            myKeyStore.load(CTJS::class.java.getResourceAsStream("/ctjskeystore.jks"), "changeit".toCharArray())
+            val kmf = KeyManagerFactory.getInstance(KeyManagerFactory.getDefaultAlgorithm())
+            val tmf = TrustManagerFactory.getInstance(TrustManagerFactory.getDefaultAlgorithm())
+            kmf.init(myKeyStore, null)
+            tmf.init(myKeyStore)
+            var ctx = SSLContext.getInstance("TLS")
+            ctx?.init(kmf.keyManagers, tmf.trustManagers, null)
+            ModuleManager.generalConsole.println("SSL Context Initialized")
+            ctx
+        } catch (e: Exception) {
+            ModuleManager.generalConsole.println("Failed to load keystore. Web requests may fail on older Java versions", LogType.WARN)
+            ModuleManager.generalConsole.println(e.stackTrace, LogType.ERROR)
+            null
+        }
+
+    }
+    fun getSSLContext(): SSLContext? {
+        return sslContext;
+    }
```

Comment:
This isn't necessary, the field isn't used outside of the class

Reply:
I added this function so modules would be able to use the up to date SSL Context in their requests if they wanted, without being able to modify it or needing to manage it themselves. I can remove it if you don't think it should be there though.

Reply:
I think I'd rather keep this an implementation detail, the `makeWebRequest` function is already public so they can use that to get access to the functionality

Reply:
False brought up a good point that axios and requestv2 will probably need access to this. However, there's still no need to make it its own function, just make the field public. And you might as well add a `@JvmStatic` annotation while you're at it.